openapi/adp.workerdemographics/Dependencies.toml

```diff
@@ -0,0 +1,9 @@
+[[dependency]]
+org = "ballerina"
+name = "http"
+version = "2.0.0"
+
+[[dependency]]
+org = "ballerina"
+name = "url"
+version = "2.0.0"
```

Comment:
Add auto-generated Dependencies.toml generated using RC6 pack or later version than that